src/utils.js

```diff
@@ -193,6 +196,8 @@ export const setSendOnlyIfErrors = () =>
 export const setPruneTraceOff = () =>
   (process.env[PRUNE_TRACE_OFF_FLAG] = 'TRUE');
 
+export const setStoreLogsOn = () => (process.env[STORE_LOGS_FLAG] = 'TRUE');
```

Comment:
Don't we need a clean function for the tests?

Reply:
Not sure I understand, but the test infra here take care of cleaning the env vars each test